### PR TITLE
Changed login view to align password field

### LINKED ISSUE #7008: Login view password field
In the login view, the password field eye icon is not aligned properly.

Screenshot 2019-12-19 at 00 01 45

### What's your PublicLab.org username?
emirvatric

Chrome - Version 79.0.3945.88 (Official Build) (64-bit)

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site. 

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by EmirVatric**
Fixes #7008

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

Screenshot 2019-12-19 at 00 01 45
Screenshot 2019-12-19 at 00 27 30

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

### CODE REVIEW

**Comment by EmirVatric** [comment]
@publiclab/reviewers
First time contributing, can You please tell me if I am doing this right. Thank you ✌️

File: app/views/user_sessions/_form.html.erb [hunk 1]
```diff
@@ -1,110 +1,104 @@
...
-              class="fa fa-eye input-group-text"
-              aria-hidden="true">
-            </a>
-          </div>
+    <div class="form-group has-feedback">
+      <label class="control-label" for="password"><%= t('user_sessions.new.password') %></label>
+      <div class="input-group">
+        <%= f.password_field :password, { tabindex: 2,placeholder: "Password",class: 'form-control', id: 'password-signup', onpaste: 'return false;' , required: true } %>
+        <div class="input-group-append">
+          <a id="visibility" class="fa fa-eye input-group-text d-flex align-items-center" aria-hidden="true">
```
Comment (VladimirMikulic) [inline app/views/user_sessions/_form.html.erb]: The real change here is that `d-flex` class has been added to the `a#visibility` element.
I can confirm that this works and solves misalignment problem on all pages.

**Comment by SidharthBansal** [comment]
Can someone remove the conflicts?
So that I can merge
Thanks all

**Comment by VladimirMikulic** [comment]
@EmirVatric are you able to resolve conflicts on your own?
If not please comment and I'll help you.

**Comment by VladimirMikulic** [comment]
Excellent work @EmirVatric.
@SidharthBansal it's ready :rocket:

**Comment by EmirVatric** [comment]
@VladimirMikulic I have resolved the conflict since PR was opened we had other pr being merged, I have tested it on local env, if You could test it as well it would be great :)

**Comment by VladimirMikulic** [comment]
@EmirVatric I know. I've already tested it. Bravo :)

File: app/views/user_sessions/_form.html.erb (hunk 1 above)
Comment (SidharthBansal) [reply app/views/user_sessions/_form.html.erb]: Is this solved?

Comment (VladimirMikulic) [reply app/views/user_sessions/_form.html.erb]: Yes, it is.

**Pull request merged**